miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppMessageBridge.kt

```diff
@@ -16,7 +18,7 @@ import com.rakuten.tech.mobile.miniapp.permission.MiniAppPermissionResult
 
 @Suppress("TooGenericExceptionCaught", "SwallowedException", "TooManyFunctions")
 /** Bridge interface for communicating with mini app. **/
-abstract class MiniAppMessageBridge {
+abstract class MiniAppMessageBridge(val activity: Activity? = null) {
```

Comment:
They already need to provide the activity context when calling `MiniApp.create`, right? So could you reuse that here? 

@khairul-alam-licon added a function for `MiniAppMessageBridge.init`, so maybe you could provide the context through that?